package.json

```diff
@@ -26,5 +26,8 @@
   "devDependencies": {
     "concurrently": "^6.3.0",
     "husky": "^7.0.4"
+  },
+  "dependencies": {
+    "iso-639-3": "^3.0.1"
```

Comment:
Could you please run `npm install iso-639-3 --prefix frontend` instead so it gets added to **./frontend/package.json**?

Reply:
[This package](https://pypi.org/project/iso-639/) is a python package used in the backend. Installed with `pip install iso-639`. Do we need to put this in frontend?

Reply:
Ohhh sorry, I didn't realize this is a backend dependency. In that case, I think this belongs to requirements.txt (**requirements.mustache**)?